examples/shader/Effect.hpp

```diff
@@ -70,7 +70,7 @@ class Effect : public sf::Drawable
     virtual void onUpdate(float time, float x, float y)                                 = 0;
     virtual void onDraw(sf::RenderTarget& target, const sf::RenderStates& states) const = 0;
 
-private:
+
```

Comment:
The `private:` and `public:` qualifiers acted as a separator between multiple groups in the class definition (e.g. methods, variables).

This is an opinionated change by a tool, not a universally accepted best practice. It means we lose that visual separation.

Reply:
I don't disagree that it's opinionated. I personally like this opinion but won't protest if others disagree.

Reply:
I'm fine with either as well, it's just a question if this change is necessary and truly improves something.
Just because a tool says so...